apps/projects/contracts/Projects.sol

```diff
@@ -578,6 +567,25 @@ contract Projects is AragonApp, DepositableStorage {
         }
     }
 
+    /**
+     * @notice Submit a fulfillment for bounty #`_bountyId` with the following info: `_data`
+     * @dev This is a noop function implemented so the client can display the radspec
+     *      for this external contract call
+     * @param _sender address of the user submitting the fulfillment
+     * @param _bountyId Standard Bounty Identifier
+     * @param _fulfillers array of users who contributed to the fulfillment of the bounty
+     * @param _data the provided proof of fulfillment
+     */
+    function fulfillBounty(
+        address _sender,
+        uint _bountyId,
+        address[] _fulfillers,
+        string _data
+    ) external
+    {
+        revert();
```

Comment:
If we just want to display the radspec, why revert? Also this should be a pure function

Reply:
Revert will cause non 0 exit status and that maybe non gracefully handled by the frontend

Reply:
This function should never execute. It's just a workaround for getting the radspec showing up in the frontend.